legacy/firmware/config.c

```diff
@@ -665,7 +665,17 @@ bool config_getLabel(char *dest, uint16_t dest_size) {
 }
 
 bool config_getLanguage(char *dest, uint16_t dest_size) {
-  return sectrue == config_get_string(KEY_LANGUAGE, dest, dest_size);
+  if (sectrue == config_get_string(KEY_LANGUAGE, dest, dest_size)) {
+    if (dest_size == 7 && (strcmp(dest, "english") != 0)) {
+      // fallthrough -> return "en-US"
+    } else {
+      // other language -> return the value
+      return true;
+    }
+  }
+  strcpy(dest, "en-US");
+  dest_size = 5;
+  return true;
```

Comment:
If `config_get_string` returns false, we still return `en-US`? Is it desired? It makes sense as a default value

Reply:
Yes, I want `en-US` to be the default even if no value is set. (We have the same behaviour in TT - we discard any values to be set and return `en-US` always).